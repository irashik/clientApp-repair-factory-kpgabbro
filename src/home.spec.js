import React from 'react';

import HomeComponent  from './home.js';


       


describe("<HomeComponent />", () => {
    test('should render correctly', () => {

        const component = shallow(<HomeComponent />)
        expect(component).toMatchSnapshot();
    });
});

