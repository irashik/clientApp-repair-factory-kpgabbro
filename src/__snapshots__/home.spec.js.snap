// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeComponent /> should render correctly 1`] = `
<Container
  fluid={true}
>
  <Row
    id="mainPage"
  >
    <h2>
      Журнал учета ремонтов оборудования.
    </h2>
  </Row>
</Container>
`;
